c/cert/test/rules/EXP40-C/test.c

```diff
@@ -0,0 +1,85 @@
+void f1() {
+  const int a = 3;
+  int *aa;
+
+  aa = &a;
+  *aa = 100; // NON_COMPLIANT
+}
+
+void f1a() {
+  const int a = 3;
+  int *aa;
+
+  aa = &a; // COMPLIANT
+}
+
+void f2() {
+  int a = 3;
+  int *aa;
+  a = 3;
+
+  aa = &a;
+  *aa = a;
+  *aa = &a;
+}
+
+void f4a(int *a) {
+  *a = 100; // NON_COMPLAINT
+}
+
+void f4b(int *a) {}
+
+void f4() {
```

Comment:
These were non-compliant in the original test case. Do you recall why they were switched to compliant?